tests/cases/distributed/explain/analyze.result

```diff
@@ -19,6 +19,24 @@ VALUES
 
 Affected Rows: 3
 
+-- SQLNESS REPLACE (metrics.*) REDACTED
+-- SQLNESS REPLACE (RoundRobinBatch.*) REDACTED
+-- SQLNESS REPLACE (-+) -
+-- SQLNESS REPLACE (\s\s+) _
+-- SQLNESS REPLACE (peers.*) REDACTED
+-- SQLNESS REPLACE region=\d+\(\d+,\s+\d+\) region=REDACTED
+explain analyze SELECT * FROM system_metrics where host in ('host1');
```

Comment:
I can also add a case to select the results.
```
SELECT * FROM system_metrics where host in ('host1');

-- test explain analyze...
explain analyze...
```

We can move this test to the `prune.sql` file, after this case.
https://github.com/GreptimeTeam/greptimedb/blob/3d1b8c4fac9bf2252213a20eceb8c95104f22dd2/tests/cases/standalone/common/select/prune.sql#L23-L29

Reply:
@evenyag 

I addressed your review.